code/modules/events/wizard/ghost.dm

```diff
@@ -17,7 +17,7 @@
 	weight = 2
 	typepath = /datum/round_event/wizard/possession
 	max_occurrences = 5
-	earliest_start = 0
+	earliest_start = 0 MINUTES
```

Comment:
why

Reply:
In case someone makes a future PR that increases the time to >0